### PR TITLE
I18N: Localise language names in GUI

### PR DESCRIPTION
**Pull request by BenCastricum**
This approach is based on placing the language names in a separate .po(t) file. Only for languages which are compatible with the current GUI limitations (Latin-1) a .po file should be created.

### CODE REVIEW

**Comment by sev-** [comment]
I think it is not quite correct concept.

The problem is that I will have difficulties with switching back to my language from, say, Japanese. E.g. I have no clue that ロシア語 means "Russian language", while I still will recognize it in English.

**Comment by BenCastricum** [comment]
True, if you change to GUI to a language you can't read, then it's hard to use it. But that's an issue that is not ScummVM specific, anything that supports multiple languages has this problem. Deleting the .ini might be suggested as a solution. Or perhaps ScummVM should get a safe-mode CLI switch?
Do you want me to submit a "safe-mode" patch? Or do you consider this a a blocking issue anyway in which case I'll cancel this pull request. I can't solve the issue you describe when the goal is "localize everything".

**Comment by rootfather** [comment]
I'm a fan of localizing everything. A safe-mode CLI switch would be quite helpful to address the problemn sev- mentiones. However, nearly every program with foreign language support has the problem that you might end up with a screwed user experience if you choose a language you can't read.

**Comment by sev-** [comment]
@BenCastricum a safe mode CLI setting sounds good though it could be not very usable, particularly for the Windows users. Do you have any better ideas on how to avoid a 'language lock-in"?

**Comment by BenCastricum** [comment]
How does "hold shift on startup" sound as a way to activate safe mode (which is what my email client uses too)?  This might be doable with ScummVM as well. Of course this doesn't work on keyboardless setups. But then again, it already is a corner case situation what we are trying to solve here.

**Comment by rootfather** [comment]
Sev, I don't see why a specific CLI command should be problematic for Windows users as long as we cover this in the README? Would be the easiest way to prevent a lock-in and should be easier to implement than a safe-key that has to be hold on program startup.

**Comment by BenCastricum** [comment]
Actually the situation sev describes will never happen with this proposal since we only support ascii. GUI / Port limitations prevent the use of non-ascii game descriptions.

File: gui/launcher.cpp [hunk 1]
```diff
@@ -88,6 +88,524 @@ enum {
 	kCmdSavePathClear = 'PSAC'
 };
 
+<<<<<<< 69d43ce1e23d0e3e8d8a5826169a20a2a0cd4b33
```
Comment (sev-) [inline gui/launcher.cpp]: Ahem. This should be cleaned up.

**Comment by sev-** [comment]
I am still not in favour of this. Perhaps a separate project for making our GUI UTF-8 could help making it properly. At this moment, however, the approach has too many drawbacks to be merged.

**Comment by BenCastricum** [comment]
Ok, no problem.

**Pull request closed without merging**